src/shared/lib/JsonPathUtils.ts

```diff
@@ -1,25 +1,32 @@
-export function hasJsonPathPlaceholders(message: String) {
+export function hasJsonPathPlaceholders(message: string) {
     if (message.match(/[{][$][^}]*[}]/g) === null) return false;
     return true;
 }
 
 export function replaceJsonPathPlaceholders(
-    message: String,
-    studyMetadata: any
+    message: string,
+    studyMetadata: any,
+    studyId: string
 ) {
     let placeholders = message.match(/[{][$][^}]*[}]/g);
     let placeholdersReplaced: boolean = true;
     if (placeholders !== null) {
         let jp = require('jsonpath');
         placeholders.forEach(placeholder => {
-            let placeholderReplaceValue = jp.query(
-                studyMetadata,
-                placeholder.replace(/["'{}]/g, '')
-            );
-            if (placeholderReplaceValue.length === 0)
+            let placeholderReplaceValue = '';
```

Comment:
we should probably put the require() from jsonpath at the top of the file.  not sure why it ended up down here.

Reply:
done